inst/shiny/DiagnosticsExplorer/R/ResultRetrieval.R

```diff
@@ -424,3 +415,891 @@ getTimeDistributionResult <- function(dataSource,
     )
   return(data)
 }
+
+
+#' Returns matrix of relationship between target and comparator cohortIds
+#'
+#' @description
+#' Given a list of target and comparator cohortIds gets temporal relationship.
+#'
+#' @template DataSource
+#'
+#' @template CohortIds
+#'
+#' @template ComparatorCohortIds
+#'
+#' @template DatabaseIds
+#'
+#' @param    relationshipDays A vector of integer representing days comparator cohort
+#'           start to target cohort start
+#'
+#' @param    relationshipType What type of relationship do you want to retrieve. The
+#'           available options are 'start', 'end', 'overlap'.
+#'
+#' @return
+#' Returns a data frame (tibble)
+#'
+#' @export
```

Comment:
This code isn't part of the package so export won't do anything

Reply:
Yes